src/Resources/views/Admin/GiftCard/_form.html.twig

```diff
@@ -6,8 +6,13 @@
             {{ form_row(form.code) }}
             {{ form_row(form.enabled) }}
             {{ form_row(form.expiresAt) }}
-            <div class="ui info message">{{ 'setono_sylius_gift_card.ui.customer_help_text'|trans }}</div>
-            {{ form_row(form.customer) }}
+            {{ form_row(form.customer, {'attr': {'data-target': '.js-ssgc-send-customer-email'}}) }}
+            {% if form.sendNotificationEmail is defined %}
+                <div class="js-ssgc-send-customer-email" style="height: 0; visibility: hidden;">
```

Comment:
```suggestion
                <div class="js-ssgc-send-customer-email" style="display:none">
```
?

Reply:
Display none will make it disappear from the DOM and it may lead to undesired side effects as the form value would not be submitted at all. So I prefer this approach

Reply:
Display none will make it disappear from the DOM and it may lead to undesired side effects as the form value would not be submitted at all. So I prefer this approach

Reply:
Yes, but we also have a default value on the entity right, so this is actually what we want, right?